test/definitions/ohi/linux/elasticsearch-debian.json

```diff
@@ -0,0 +1,55 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+  
+    "resources": [{
+        "id": "host1",
+        "display_name": "Debian10InfraInstallHost",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.small",
+        "ami_name": "SupportedImages debian-10-amd64-*",
+        "user_name": "admin"
+    }],
+  
+    "services": [{
+      "id": "elastic1",
+      "destinations": ["host1"],
+      "source_repository": "https://github.com/newrelic/open-install-library.git",
+      "deploy_script_path": "test/deploy/linux/elasticsearch/install/debian/roles",
+      "port": 80
```

Comment:
@setamyDG if port 80 required to be opened? If not use the port value of 9999 (internal skip).